### PR TITLE
Fixes #1091 : Changes Nav-bar : comply with eventyay

### LINKED ISSUE #1091: Changing the NavBar
**I'm submitting a ...**
- [ ] bug report
- [X] feature request

Change the navbar to reflect the style of eventyay.

**I am sending a PR for the same**

### PR DESCRIPTION
**Pull request by gabru-md**
Fixes #1091 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] I have added necessary documentation (if appropriate)

### Preview Link 

- **Replace XXX with your PR no**
- Link to live demo: http://pr-XXX-fossasia-badgeyay.surge.sh  

#### Changes proposed in this pull request:
- Screenshots below 
- 1
prevheader

- 2
newpage

- 3
event_badge

### CODE REVIEW

File: frontend/app/templates/components/nav-bar.hbs [hunk 1]
```diff
@@ -1,19 +1,28 @@
-<div class="ui top fixed menu">
+<div class="ui top absolute menu">
```
Comment (ParthS007) [inline frontend/app/templates/components/nav-bar.hbs]: It can work without `absolute`.

File: frontend/app/styles/partials/welcome.scss [hunk 2]
```diff
@@ -18,14 +18,16 @@ $color4: #ffa500;
...
     .lead {
-      font-size: 65px;
+      font-size: 3.5rem;
+      font-weight: 300 !important;
       text-shadow: 0 1px 2px $color2;
     }
 
     .create-badges {
-      font-weight: 800;
+      font-weight: 400 !important;
```
Comment (ParthS007) [inline frontend/app/styles/partials/welcome.scss]: no need of `!important`

File: frontend/app/styles/partials/welcome.scss [hunk 3]
```diff
@@ -18,14 +18,16 @@ $color4: #ffa500;
     right: 20%;
     text-align: center;
     top: 25vh;
+    vertical-align: center;
 
     .lead {
-      font-size: 65px;
+      font-size: 3.5rem;
+      font-weight: 300 !important;
```
Comment (ParthS007) [inline frontend/app/styles/partials/welcome.scss]: no need of `!important`

**Comment by gabru-md** [comment]
overriding is needed. Can we just ignore codacy for once? :sweat_smile:

**Comment by djmgit** [comment]
@gabru-md please resolve conflicts.

**Comment by gabru-md** [comment]
@djmgit @ParthS007 @yashLadha please merge

File: frontend/app/styles/partials/welcome.scss [hunk 4]
```diff
@@ -18,14 +18,16 @@ $color4: #ffa500;
     right: 20%;
     text-align: center;
     top: 25vh;
+    vertical-align: center;
 
     .lead {
-      font-size: 5em;
+      font-size: 3.5rem;
```
Comment (yashLadha) [inline frontend/app/styles/partials/welcome.scss]: Can you change it to `4em` as it will be small on higher resolution

**Comment by gabru-md** [comment]
done @yashLadha

**Pull request merged**